api/websocket/server/server.go

```diff
@@ -570,3 +625,19 @@ func (ws *WsServer) sendInboundRelayMessageToClient(v interface{}) {
 		log.Error("Decode relay message failed")
 	}
 }
+
+// client auth, generate challenge
+func (ws *WsServer) sendClientAuthChallenge(sess *session.Session) error {
+	resp := make(map[string]interface{})
```

Comment:
Don't make your own response here, use `ResponsePack()`. You can find examples in this file

Reply:
yes. fixed.